src/solver/optimisation/adequacy_patch_csr/hourly_csr_problem.h

```diff
@@ -64,6 +65,10 @@ class HourlyCSRProblem
     void setQuadraticCost();
     void setLinearCost();
 
+private:
+    using AdqPatchParams = Antares::Data::AdequacyPatch::AdqPatchParams;
+    AdqPatchParams& adqPatchParams_;
```

Comment:
```suggestion
    const AdqPatchParams adqPatchParams_;
```
Copy to avoid side effects, that is :
1. Us changing values in `adqPatchParams_` (can be avoided by adding a `const` qualifier to the existing reference)
2. Others changing values in `adqPatchParams_` (can't be avoided, but shoudn't happen)

Reply:
You can use a `const ref` if you prefer

Reply:
Done